Composer/packages/tools/language-servers/language-generation/src/LGServer.ts

```diff
@@ -553,6 +559,32 @@ export class LGServer {
       return Promise.resolve(null);
     }
 
+    const wordRange = getEntityRangeAtPosition(document, params.position);
+    const word = document.getText(wordRange);
+    const projectId = this.getLGDocument(document)?.projectId;
+    let luContents: string[] = [];
+    if (projectId && this.entitiesResolver) {
+      luContents = this.entitiesResolver(projectId) || [];
```

Comment:
Can you fetch this data from server only if your time check on L#572 allows it?

Reply:
changed the location of fetch lu files contents